tools/docker/builder/openwrt/profiles_feeds/netgear-rax40.yml

```diff
@@ -0,0 +1,30 @@
+  - name: feed_target_mips
+    uri: https://intel.prpl.dev/prplmesh/feed_target_mips.git
```

Comment:
I think we should keep our stuff on git.prpl.dev. Or is it difficult to fork it there?

Is there any way to get these changes upstream?

Reply:
It's not entirely clear to me, but I have the impression that intel.prpl.dev is a completely separate Gitlab instance, which was cloned from git.prpl.dev at some point.

I wasn't able to fork intel.prpl.dev/intel to git.prpl.dev/prplmesh, but I saw that some of our git.prpl.dev/prplmesh forks were at intel.prpl.dev/prplmesh already...